prework.md

```diff
@@ -0,0 +1,57 @@
+| Table Of Contents                   |
```

Comment:
add steps for windows 
- [ ] git bash install [git for windows](https://gitforwindows.org/)